README.md

```diff
@@ -1,4 +1,5 @@
-# Neptune documentation
+# Neptune legacy-docs
```

Comment:
I would keep the names "legacy documentation" or sth like this.

Reply:
```suggestion
# Neptune legacy-docs
```

Reply:
```suggestion
# Neptune legacy documentation
```

Reply:
@Herudaio fixed